### ISSUE TITLE
Feature request: Discord webhook for Form

### ISSUE THREAD

**Issue by Pandexe** [pojome/elementor#4218]
## Description

Get the discord webhook to work just as the other services integrate in form plugin. 

## Steps to reproduce
Get the discord webhook link
Set it in to the webhook form

It will fail when submitting the form. 

## Environment
wordpress. 
Elementor pro.

**Comment by bainternet** [comment]
Discord webhooks don't know how to collect and display the data from Elementor's webhook.

But you can set up a custom webhook from your site fairly easily and get notified in discord on each submission:
![screen shot 2018-04-24 at 15 33 18](https://user-images.githubusercontent.com/660572/39187060-ea349798-47d4-11e8-8669-0120c624db30.png)


```PHP
add_action( 'elementor_pro/forms/new_record', function( $record, $handler ) {
	//make sure its our form
	$form_name = $record->get_form_settings( 'form_name' );
	if ( 'MY_FORM_NAME' !== $form_name ) {
		return;
	}

	$page_url = isset( $_POST['referrer'] ) ? $_POST['referrer'] : site_url();
	$icon = 'https://i.imgur.com/N0tCpAI.png';

	// prepare Form Data
	$raw_fields = $record->get( 'fields' );
	$fields = [];
	foreach ( $raw_fields as $id => $field ) {
		$fields[] = [
			'name'   => $id,
			'value'  => $field['value'],
			'inline' => true,
		];
	}

	// Build discord  webhook data
	$webhook_data = [
		'username' => 'Elementor Bot',
		'avatar_url' => $icon,
		'content' => 'A new Form Submission',
		'embeds' => array_values( [
			[
			'author'      => [
				'name'     => 'Elementor',
				'url'      => $page_url,
				'icon_url' => $icon,
			],
			'fields'      => array_values( $fields ),
			'title'       => 'Elementor Form',
			'url'         => $page_url,
			'description' => 'A new Form Submission',
			'color'       => hexdec( '9c0244' ),
			'thumbnail'   => [
				'url' => $icon,
			],
			'image'       => [
				'url' => $icon,
			],
			'footer'      => [
				'text'     => 'Woah! So cool! :smirk:',
				'icon_url' => $icon,
			],
		],
		]),
	];

	$body = json_encode( $webhook_data );
	wp_remote_post( 'https://discordapp.com/api/webhooks/YOUR-WEBHOOK-URL-HERE', [
			'body' => $body,
	]);
	return;
}, 10, 2 );
```

**Comment by Pandexe** [comment]
Thank you. I managed to get it working though iffft as a workaround.

Will try This later to se what works Best.
Thanks.

Den tir. 24. apr. 2018 14.34 skrev Raz Ohad <notifications@github.com>:

**Comment by arielk** [comment]
This integration is now available in Elementor Pro v2.3.0

Thanks for your feedback!

**Comment by clibarnes** [comment]
> This integration is now available in Elementor Pro v2.3.0
> 
> Thanks for your feedback!

not working as of 2021 Aug in the current version

**Comment by iH4xz** [comment]
I would like to ask you add 'Content' so I can mention roles and more things like notes and so on.
